frontend/src/components/DragAndDrop/DragAndDrop.css

```diff
@@ -0,0 +1,29 @@
+#form-file-upload {
```

Comment:
I think it's generally more preferred to apply styles to classes instead of ids, but that's not a big deal